types/dependency-tree/index.d.ts

```diff
@@ -1,20 +1,39 @@
-// Type definitions for dependency-tree 7.2
+// Type definitions for dependency-tree 8.0
 // Project: https://github.com/mrjoelkemp/node-dependency-tree
 // Definitions by: Joscha Feth <https://github.com/joscha>
 //                 Alex <https://github.com/adjerbetian>
+//                 Christian Rackerseder <https://github.com/screendriver>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.5
+
+import { CompilerOptions } from 'typescript';
 
 declare namespace dependencyTree {
     interface DependencyObj {
         [k: string]: DependencyObj;
     }
 
+    interface TsConfigCompilerOptions extends Omit<CompilerOptions, 'module' | 'target'> {
+        module?: 'None' | 'CommonJS' | 'AMD' | 'System' | 'UMD' | 'ES6' | 'ES2015' | 'ES2020' | 'ESNext';
+        target?: 'ES3' | 'ES5' | 'ES6' | 'ES2015' | 'ES2016' | 'ES2017' | 'ES2018' | 'ES2019' | 'ES2020' | 'ESNext';
+    }
+
+    interface TsConfig {
+        extends?: string;
+        compilerOptions: TsConfigCompilerOptions;
+        exclude?: string[];
+        include?: string[];
+        files?: string[];
+        compileOnSave?: boolean;
+        references?: Array<{ path: string }>;
+    }
```

Comment:
Typing this as precisely here seems a bit extra to me. I do not feel it brings so much value to know what the TsConfig interface looks like here. And anytime the `TsConfig` interface would change, this code would have to be updated.

I think I would just go for `tsConfig?: string | Record<string, any>;`.

Wdyt?

Reply:
We can do that, yes. I did not do it because I never use `any` for anything 😅 

Done in 0404b4c325ddabe151d36a6b3f959cf58e322566

Reply:
@screendriver :+1: 
I actually agree that using `any` is not the most elegant :smile:, but in this special case, it feels like a good trade off.